prometheus/grr.libsonnet

```diff
@@ -0,0 +1,56 @@
+/*
+ * The `grr` function below will render prometheus resources generated by this
+ * prometheus library (e.g. in either `prometheusRules` or `prometheusAlerts`)
+ * into Kubernetes style objects, with kind/apiVersion. (Grizzly)[https://github.com/grafana/grizzly)
+ * has the ability to consume resources in this format.
+ *
+ * It is invoked with a different entrypoint and handed your jsonnet as input. E.g
+ * if your main entrypoint is `main.jsonnet`, this can be invoked like so:
+ *
+ * grr.jsonnet:
+ * ```
+ * local prom = import 'prometheus/grr.libsonnet';
+ * prom.grr(import 'main.jsonnet')
```

Comment:
The API could be something like his, not grabbing things it likes in out of root, rather allowing someone to feed in mixins. It could be named differently or be nested somewhere:

```jsonnet
local grr = import 'grizzly/grr.libsonnet';
local main = import 'main.jsonnet';

grr.prometheus.fromMixins(main.mixins)
+ grr.prometheus.fromRules(main.prometheusRules)
```

Reply:
looks good